### ISSUE TITLE
Winterfell: Post code 0x0d after CMOS clear

### ISSUE THREAD

**Issue by osresearch** [linuxboot/linuxboot#17]
![CMOS clear jumper](https://farm1.staticflickr.com/792/40653414814_f28f577b50_z_d.jpg)

If jumper J43 is moved to pins 2-3 for a few seconds the CMOS NVRAM will be cleared.  On reboot the AMI DxeCore will halt the system with post code 0x0d during the call to `ExitBootServices()`, possibly indicating a segfault.

Reflashing with the stock AMI BIOS and rebooting will re-write the contents of the NVRAM and allow LinuxBoot to be reflashed.  The contents of the CMOS NVRAM were the same after two repetitions of this process:

```
# xxd -g4 /sys/devices/pnp0/00\:04/nvram
00000000: 00000001 d220827f 02ffff02 43548a1b  ..... ......CT..
00000010: 0013e641 82920012 17004306 0ca94202  ...A......C...B.
00000020: 08ebffff 20040cf0 07020000 00000000  .... ...........
00000030: 00000000 01100128 02028180 01000041  .......(.......A
00000040: 01c86040 d1400000 02200480 04200000  ..`@.@... ... ..
00000050: 0000a420 c0002008 82020000 9400424a  ... .. .......BJ
00000060: 9200a060 08402202 0010010a 80502080  ...`.@"......P .
00000070: 0200                                 ..
```

**Comment by osresearch** [comment]
Including `CmosManagerSmm` does not fix it -- clear CMOS still results in 0x0d.

(The deps for this module might not be right -- if it is before the SMM in the DXE the system crashes silently after bds_main() but before efi_main())

**Comment by osresearch** [comment]
Including `PostReport` (`5446c293-339b-47cd-b719-585de39408cc`) seems to allow it to recover from the clear CMOS contents.

**Comment by osresearch** [comment]
Fixed as part of #19
